src/test.rs

```diff
@@ -13,6 +13,14 @@ use std::char;
 use super::UnicodeNormalization;
 use super::char::is_combining_mark;
 
+fn to_codepoint_seq(s: String) -> String {
+    s.chars().map(|c| format!("U+{:04X} ", c as u32)).collect::<String>()
+}
+macro_rules! assert_eq_strs {
```

Comment:
does this strip out the combining grapheme joiner?  if so, we're not testing that we didn't regress the non stream safe version, yeah?